Libraries/Image/RCTImageView.mm

```diff
@@ -86,6 +86,10 @@ - (instancetype)initWithBridge:(RCTBridge *)bridge
 {
   if ((self = [super initWithFrame:CGRectZero])) {
     _bridge = bridge;
+    _imageView = [[RCTUIImageViewAnimated alloc] init];
+    _imageView.autoresizingMask = UIViewAutoresizingFlexibleWidth | UIViewAutoresizingFlexibleHeight;
```

Comment:
What's happening on this line?

Reply:
I don't know. I just moved some lines up to group things together.